### DISCUSSION TITLE
View existing user photo

### DISCUSSION THREAD

**Discussion by anchovy** [solspace/craft-freeform#1050] [Questions & Answers] [answered]
We've got a 'edit details' form and we're mapping an image upload to the users photo field.
Is there a way to display the current photo in the form?

I've tried adding a html field with allow twig and using the code:
```
{% if currentUser %}
    {% if currentUser.photoUrl %}
        <img src="{{ currentUser.photoUrl }}" alt="User Photo">
    {% else %}
        <p>No photo available for the current user.</p>
    {% endif %}
{% else %}
    <p>User is not logged in.</p>
{% endif %}

```

But all I'm getting is 'User is not logged in.' message

Thanks in advance.

**Comment by kjmartens** [comment, marked as answer]
Hi @anchovy,

The problem here is that the Twig is likely being rendered before the currently logged in user data is available, thus always evaluating to false. You may need to approach this a different way... either loading that code and conditional outside of the form or maybe coding your form template code directly inside the regular template (instead of using a formatting template). 🙂
